backend/src/db-utils.js

```diff
@@ -120,51 +84,173 @@ const queryOnlineNodes = async () => {
   ]);
 };
 
+//new query for new dashboard
+const queryDashboardBlockInfo = async (options) => {
+  async function queryLatestBlockHeight({ dataSource }) {
+    let query;
+    if (dataSource === DS_INDEXER_BACKEND) {
+      query = `SELECT block_height FROM blocks ORDER BY block_height DESC LIMIT 1`;
+    } else {
+      query = `SELECT height AS block_height FROM blocks ORDER BY height DESC LIMIT 1`;
+    }
+    return await querySingleRow([query], { dataSource });
+  }
+  async function queryLatestGasPrice({ dataSource }) {
+    let query;
+    if (dataSource === DS_INDEXER_BACKEND) {
+      query = `SELECT gas_price FROM blocks ORDER BY block_height DESC LIMIT 1`;
+    } else {
+      query = `SELECT gas_price FROM blocks ORDER BY height DESC LIMIT 1`;
+    }
+    return await querySingleRow([query], { dataSource });
+  }
+  async function queryLastMinuteBlocks({ dataSource }) {
+    let query;
+    if (dataSource === DS_INDEXER_BACKEND) {
+      query = `SELECT block_timestamp AS latest_block_timestamp FROM blocks ORDER BY block_timestamp DESC LIMIT 1`;
+    } else {
+      query = `SELECT timestamp AS latest_block_timestamp FROM blocks ORDER BY timestamp DESC LIMIT 1`;
+    }
+    const latestBlockTimestampOrNone = await querySingleRow([query], {
+      dataSource,
+    });
+    if (!latestBlockTimestampOrNone) {
+      return { total: 0 };
+    }
+    const {
+      latest_block_timestamp: latestBlockTimestamp,
+    } = latestBlockTimestampOrNone;
+    const latestBlockTimestampBN = new BN(latestBlockTimestamp);
+    const nowEpochTimeBN = new BN(Math.floor(new Date().getTime() / 1000));
```

Comment:
After a second read, I think I was not accurate enough here; please, rename `nowEpochTimeBN` to `nowUnixTimeBN` to avoid confusion with the NEAR epochs, and apply the same reasoning to the rest of the "EpochTime" variables.

Reply:
change to `currentunixTimeBN`